proc_nutation.py

```diff
@@ -25,8 +25,8 @@
     # in particular -- if you don't do this before convolution, the
     # convolution doesn't work properly!
     s.ift('t2')
-    # {{{ centering of data using hermitian function test
-    else:
+    if 'amp' not in s.dimlabels:
+        # {{{ centering of data using hermitian function test
```

Comment:
make sure you've checked this program -- the hanging else you had would have given a syntax error.

Reply:
I've run now and it does work! good catch!